manim/_config/utils.py

```diff
@@ -542,15 +543,23 @@ def digest_parser(self, parser):
             setattr(self, key, parser["CLI"].get(key, fallback="", raw=True))
 
         # float keys
-        for key in ["background_opacity"]:
+        for key in [
+            "background_opacity",
+            # the next two are floats but have their own logic, applied later
+            # "frame_width",
```

Comment:
Is this supposed to be commented?

Reply:
I don't understand what you mean here.

Reply:
I see a list that only iterates over one entry called background_opacity, so I ask myself what the purpose if this for loop might be

Reply:
I see. The loop is there for symmetry with the other loops. In theory, this part of the code should loop over every config option that is a float, and execute the same code for all of them. It just so happens that, in the past we only had one option that is a float.

Now, in this PR I am adding two config options that are floats and *should* be added to that loop. HOWEVER, they have their own logic, so their not being in the loop is actually an exception to the rule. So I decided to add them but commented out, as a way of signaling that "these should be here but we must process them later because they are an exception". 

Perhaps I could change the comment on L548 of this PR to clarify this.